oojs.js

```diff
@@ -1,12 +1,21 @@
-// Define a Playlist class. It should have...
-  // A title property that is determined by some input.
-  // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
-  // A favorites property that is initialized at 0. This cannot be set by user input.
-  // An addSong method that adds a song to the songs property.
+class Playlist {
+  constructor(title, song1, song2) {
+  this.title = title;
+  this.songs = [song1, song2];
+  this.favorites = 0;
+  var addSong = function(song) {
```

Comment:
While the function itself looks good, you're not attaching this method to the class (i.e., you're not able to call `playlistInstance.addSong("song title")`.

What you need to do is make `addSong` a method separate from the constructor in the class definition, like so...

```js
class Playlist {
  constructor(title, songs){
    // assign properties
  }
  addSong(song){
    this.songs.push(song);
  }
}
```